app/internal/global_variable.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import Request
+
+from app.dependencies import templates
+from app.internal.security.ouath2 import (
+    Session, get_jwt_token, get_authorization_cookie
+)
+
+
+async def get_user_for_global_var(db: Session, jwt: str) -> str:
+    jwt_payload = await get_jwt_token(db, jwt)
```

Comment:
This will make the site query the database with every request to the site.
It's nullify the advantages of JWT

Reply:
So maybe I could pull the token out of the cookie, and if the token is in the right length (and maybe more tests), I wiil set a global var: 'is_token'.
To verify that there is a logged in user. 
Tell me what you think

Reply:
Great! You can also take the user details from the JWT, not from the database.